README.md

```diff
@@ -26,6 +27,43 @@ Bei Fragen und Anregungen entweder ein Issue in GitHub anlegen oder an devsuppor
 * [Vorherige Releases](#vorherige-releases)
 * [Nutzungsbedingungen](#nutzungsbedingungen)
 
+## Die HTTP Schnittstelle
+
+Die HTTP Schnittstelle der PEX-API ist unter der Basis-Url
+```
+https://api.europace.de/v2/partner/
+```
+erreichbar.
```

Comment:
Ergänze: 

Die PEX 2 API unterstützt das Datenformat openAPI 3 <Link zur yaml>. Unter <Link zu OpenAPI Codegen> kann man sich einen Client in seiner bevorzugten Programmiersprache generieren lassen:
"Code Snippet Java"

Reply:
https://openapi-generator.tech/

Reply:
https://swagger.io/tools/swagger-codegen/

Reply:
Verweis auf OpenAPI und Swagger.io ist eingebaut.